src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -1,3 +1,9 @@
+<Callout>
+
+SSR functionality in Amplify was primarily built for NextJS support and we cannot guarantee support on other frameworks. Additionally, it [does not work in a middleware context](https://github.com/aws-amplify/amplify-js/issues/9145) as it would in getServerSideProps.
```

Comment:
Is it common process in our docs to cite github issues?

Reply:
@abdallahshaban557 Instead of saying calling out a GH issue to state what this _doesn't_ work with, do we want to say something like this? 

SSR functionality in Amplify was primarily built for compatibility with NextJS [page components](https://nextjs.org/docs/basic-features/pages), and their [getServerSideProps data fetching mechanism](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props). Compatibility with other frameworks or NextJS features cannot be guarenteed.

Reply:
I love that suggestion @haverchuck  - @tannerabread , let us please make that change!

Reply:
got it, thanks for the suggestion @haverchuck !